prow/deck/OWNERS

```diff
@@ -1,2 +1,2 @@
 labels:
- - area/prow/deck
+ - area/prow/deck
```

Comment:
nit: newline? /shrug

Reply:
not worth bothering for a re-LGTM ;)

Reply:
I'm quick to re-lgtm generally, and flip-flopping newlines is 👎 
unix files or bust! 😛